flink-staging/flink-table/pom.xml

```diff
@@ -59,6 +59,12 @@ under the License.
 		</dependency>
 
 		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-hcatalog</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<dependency>
```

Comment:
I meant putting the HCat specific stuff into a different package. But I now realize that we could not have the `fromHCat` call on `TableEnvironment` then. So this is ok, I guess.